backend/eventd/silenced_test.go

```diff
@@ -187,6 +187,7 @@ func TestHandleExpireOnResolveEntries(t *testing.T) {
 	resolution := func(e *types.Event) *types.Event {
 		e.Check.History = []types.CheckHistory{
 			types.CheckHistory{Status: 1},
+			types.CheckHistory{Status: 0},
```

Comment:
No, because this test assumed the last element in the check history was the _previous_ check result and not the current one, which isn't the case. And it was using the check status as the current one.